docs/templates/eia861_child.rst.jinja

```diff
@@ -36,4 +70,6 @@ details, see the `official EIA-861 page <{{ source.path }}>`__
 * We also have not yet integrated the Balancing Authority and Utility IDs reported in
   the EIA-861 into our entity tables, so for the moment we don't have any foreign key
   constraints enabled on the EIA-861 tables.
+* We have not yet integrated the 2018 EIA-861S data. Details about this are
+  mentioned in issue :issue:`3654`
```

Comment:
This isn't quite right. I'd say:

"In 2019 the data collected from the short form was incorporated into the other published tables rather than as a standalone Short_Form_####.xlsx (where #### is the year) as in other years. We haven't addressed this discrepancy yet; see details in issue :issue:`3654`.